packages/types/src/frontmatter.ts

```diff
@@ -321,7 +326,7 @@ export interface Frontmatter extends TransitionOptions {
    *
    * See https://sli.dev/features/draggable
    */
-  dragPos?: Record<string, string>[]
+  dragPos?: Record<string, string> | Record<string, string>[]
```

Comment:
I think this should be just `Record<string, string>`:

```suggestion
  dragPos?: Record<string, string>
```

Reply:
@kermanx I think so too, and I use the object myself, but through the array we can also use it 👀